package.json

```diff
@@ -37,7 +37,7 @@
     "test": "jest --reporters=default",
     "test:smoketests": "nyc node smoketests",
     "test:coverage": "nyc --no-clean --require ts-node/register jest",
-    "test:cli": "jest test --reporters=default",
+    "test:cli": "jest test --reporters=default --forceExit",
```

Comment:
it is very bad, it means you don't close something, we need to fix code/test

Reply:
Yup, it is bad. Had looked into it earlier as well, I came across a bug in `execa` itself, will be exploring more.

Reply:
Removed it for now.